Automattic/wp-calypso
# Create Stepper flow foundation

#### Changes proposed in this Pull Request

* This PR adds a stepper flow next to `gutenboarding` flow and reuse whatever possible from Gutenboarding.
* I wanted to make `useSteps` and `useStepNavigation` into packages, but they seem to be very particular to every individual flow. Make we can make a flow factory that accepts these hooks and create a flow around them. 


#### Testing instructions

1. Run Calypso.
2. Go to /stepper.
3. You should be redirected to `/stepper/first`.
4. You should be able to move between the steps using the buttons provided and the browser back/forward buttons too.

For context, see: pdDR7T-1B-p2
